app/src/main/java/ch/epfl/sweng/SDP/home/LeaderboardActivity.java

```diff
@@ -346,7 +351,14 @@ private ValueEventListener initializeFriendsButton() {
                 @Override
                 public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                     if (dataSnapshot.exists()) {
-                        setImageResource(R.drawable.remove_friend);
+                        int status = dataSnapshot.getValue(int.class);
+                        if (status == SENT.ordinal()) {
```

Comment:
Maybe a switch here is better

Reply:
I tried to do a switch. But when i write case SEND.ordinal() (even if I put SENT.ordinal() into a final static int), I get the error "needs to be constant". Do you have an idea how to do it?